sdk-lib/Cargo.toml

```diff
@@ -10,33 +10,32 @@ publish = false
 edition = "2018"
 
 [dependencies]
-android_logger = "0.9"
+android_logger = "0.11"
```

Comment:
This was causing build issues so I reverted it a while back.
```
error: type parameters must be declared prior to const parameters
   --> /Users/carter/.cargo/registry/src/github.com-1ecc6299db9ec823/android_logger-0.11.1/src/lib.rs:437:33
    |
437 | fn uninit_array<const N: usize, T>() -> [MaybeUninit<T>; N] {
    |                -----------------^- help: reorder the parameters: lifetimes, then types, then consts: `<T, const N: usize>`

error: could not compile `android_logger` due to previous error
warning: build failed, waiting for other jobs to finish...
error: build failed
```

Reply:
I think that's due to an older Rust version; I recall the ordering being relaxed at some point. That means this dependency has an MSRV we need to take into account. But that's easy enough: we should always use the latest stable Rust version.

Reply:
Aha, it was Rust 1.59 that enabled this: https://blog.rust-lang.org/2022/02/24/Rust-1.59.0.html#const-generics-defaults-and-interleaving

We can make this easy to avoid by setting our MSRV to 1.59 in `Cargo.toml`; then the compiler will return a more useful error if it is too old.